tests/test_rotmat.py

```diff
@@ -44,6 +44,8 @@ def test_maths(self):
         assert v1 - v2 == Vector3(0, -1, -6)
         assert (v1 * 3) == Vector3(3, 6, -9)
         assert v1 * v2 == -2
+        assert v2 / 2.0 == Vector3(0.5, 1.5, 1.5)
+        assert v2 // 2.0 == Vector3(0, 1, 1)
```

Comment:
```suggestion
        assert v2 / 2.0 == Vector3(0.5, 1.5, 1.5)
        assert v2 // 2.0 == Vector3(0, 1, 1)
        assert v2 / 2.1 == Vector3(0.23809523809523808, 0.7142857142857143, 0.7142857142857143)
        assert v2 // 2.1 == Vector3(0.0, 0.0, 0.0)
```
Please add tests with non-whole number divisors.